Firmware/cardreader.cpp

```diff
@@ -183,23 +183,23 @@ void CardReader::initsd()
   {
     //if (!card.init(SPI_HALF_SPEED,SDSS))
     SERIAL_ECHO_START;
-    SERIAL_ECHOLNRPGM(_n("SD init fail"));////MSG_SD_INIT_FAIL c=0 r=0
```

Comment:
I don't think any of change of kind _n("") --> MACRO is right. _n means not to be translated, as those are messages outputted to serial line.

Reply:
Thanks for the answer...and sorry that my is so late, was on vacation.
I close this PR and try another one